pinot-core/src/test/java/org/apache/pinot/queries/JsonDataTypeQueriesTest.java

```diff
@@ -52,12 +52,12 @@
 
```

Comment:
Changed this file to be able to override index config and run for startree json index (some of the test cases depend on json_match ).